src/main/controllers/TribunalOrderOrRequestDetailsController.ts

```diff
@@ -31,10 +31,14 @@ export default class TribunalOrderOrRequestDetailsController {
     }
 
     const redirectUrl =
-      PageUrls.TRIBUNAL_RESPOND_TO_ORDER.replace(':orderId', req.params.orderId) + getLanguageParam(req.url);
+      selectedRequestOrOrder.value.sendNotificationCaseManagement === 'Request'
+        ? PageUrls.PREPARE_DOCUMENTS.replace(':orderId', req.params.orderId) + getLanguageParam(req.url)
```

Comment:
Can change the ternary to decide on the PageUrl which then can be .replaced() on, not major but could be slightly nicer

Reply:
Refactored